src/controllers/userProfileController.js

```diff
@@ -488,6 +488,31 @@ const userProfileController = function (UserProfile) {
       .catch(error => res.status(404).send(error));
   };
 
+
+  const updateOneProperty = function (req, res) {
+    const { userId } = req.params;
+    const { key, value } = req.body;
+
+    //remove user from cache, it should be loaded next time
+    cache.removeCache(`user-${userId}`);
+    if (!key || value == undefined) return res.status(400).send({error:'Missing property or value'})
```

Comment:
It's generally recommended to use strict equality `===` when checking for undefined values.

Reply:
Thank you very much for your detailed review, I commit the strict equality!